lab_1_classify_by_unigrams/main.py

```diff
@@ -61,18 +55,15 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
-    if not (
-            isinstance(predicted, list)
+    if not (isinstance(predicted, list)
             and isinstance(actual, list)
             and len(predicted) == len(actual)
     ):
         return None
 
-    sum_diff = 0
-    for freq_value in zip(predicted, actual):
-        sum_diff += (freq_value[0] - freq_value[1]) ** 2
-    mse = sum_diff / len(predicted)
-    return mse
+    summa = float(sum((p - a) ** 2 for p, a in zip(predicted, actual)))
```

Comment:
Why do you cast it to float?

Reply:
If I just use sum() without casting its result (summa) to float, then mypy says the return is Optional, and not Float which is required there.